src/server/server.ts

```diff
@@ -105,6 +160,7 @@ registerRoute('get', '/run/:host/:id', (req, res) => {
                 origin: currentConfig.editorUrl,
                 host: host,
                 assets: getAssetPaths(),
+                isTrustedSnippet: true, /* only called when snippet is already trusted */
```

Comment:
Changed this to false; heartbeat will always retrieve the updated local storage value, so should assume false until heartbeat rectifies it.